src/Avalonia.Controls/Notifications/INotificationManager.cs

```diff
@@ -13,6 +14,18 @@ public interface INotificationManager
         /// Show a notification.
         /// </summary>
         /// <param name="notification">The notification to be displayed.</param>
-        void Show(INotification notification);
+        /// <param name="cancellationToken">The cancellation token.</param>
+        void Show(INotification notification, CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Closes a notification.
+        /// </summary>
+        /// <param name="notification">The notification to be closed.</param>
+        void Close(INotification notification);
+
+        /// <summary>
+        /// Clears all notifications.
+        /// </summary>
+        void ClearAll();
```

Comment:
Why not CloseAll? To be consistent with Close method.

Reply:
https://github.com/AvaloniaUI/Avalonia/pull/15628/commits/094a3641d516b30d8757a0f3a3927e31d5254a7d